packages/styling/etc/styling.api.ts

```diff
@@ -0,0 +1,647 @@
+// @public
+export function buildClassMap<T>(styles: T): {
+    [key in keyof T]?: string;
+};
+
+// @public
+export function concatStyleSets(...styleSets: (IStyleSet<any> | false | null | undefined)[]): IConcatenatedStyleSet<any>;
+
+// @public (undocumented)
+export function createFontStyles(localeCode: string | null): IFontStyles;
+
+// @public
+export function createTheme(theme: IPartialTheme, depComments?: boolean): ITheme;
+
+// @public
+export function focusClear(): IRawStyle;
+
+// @public
+export function fontFace(font: IFontFace): void;
+
+// WARNING: Export "mini" is missing the "const" modifier. Currently the "namespace" block only supports constant variables.
+// WARNING: Export "xSmall" is missing the "const" modifier. Currently the "namespace" block only supports constant variables.

```

Comment:
Are these warnings actionable?

Reply:
Looking into it.

Reply:
🤔 aren't they marked as `const` tho? https://github.com/OfficeDev/office-ui-fabric-react/blob/6e82e75446a80acba84cc29a45d6f705c67fb828/packages/styling/src/styles/fonts.ts#L72

Interested to know how to resolve these warnings.